languages.toml

```diff
@@ -4605,3 +4605,16 @@ comment-tokens = ["#"]
 [[grammar]]
 name = "properties"
 source = { git = "https://github.com/tree-sitter-grammars/tree-sitter-properties", rev = "579b62f5ad8d96c2bb331f07d1408c92767531d9" }
+
+[[language]]
+name = "puppet"
+scope = "source.puppet"
+injection-regex = "puppet"
+file-types = ["pp"]
+roots = []
+indent = { tab-width = 2, unit = "  " }
+comment-token = "#"
+
+[[grammar]]
+name = "puppet"
+source = { git = "https://github.com/neovim-puppet/tree-sitter-puppet", rev = "bdedf4d5b13b4020745a678364e2f441cd3abf2a" }
```

Comment:
It looks like there isn't a license for this grammar. Could you suggest adding one upstream and update the revision here once it's added?

Reply:
[looks like](https://github.com/neovim-puppet/tree-sitter-puppet/blob/main/Cargo.toml#L12) the license is MIT. Would this be enough?